en/council-codex.md

```diff
@@ -0,0 +1,105 @@
+
+The Council of Elders exists to make sure this kind of support is always present — especially when it comes to our values, our vision, and the way we make big decisions together.
+
+The Council is made up of six Applifters elected by other Applifters. Its members are a cross-section of the company — from different teams, roles, and perspectives.
+
+At least one council member is always a company owner, to help ensure that strategic decisions stay in touch with the company's legal obligations.
+
+If no owner is elected, it may signal a disconnect between ownership and the rest of the company. In that case, a mediated discussion will help us explore what's behind it and how to move forward.
+
+## What the Council Does
+
+- Help C-team shape Applifting's strategy and approve it
+- Keep our mission, vision and values present in decisions
+- Provide guidance to the C-Team and oversee whether the company is staying on course
+- Be the last resort in tension and conflict resolution
+- Approve major governance topics — anything that changes how the company operates at a fundamental level
+- Approve long-term investments
+- Make sure important decisions are not made in a rush or in isolation
+
+Council members are not here to manage or micromanage. Their role is to listen, reflect, ask hard questions, and be vocal when it matters.
+
+## Relationship with the C-Team
+
+The Council and the C-Team work in close partnership. The C-Team leads the company forward day to day — setting goals, making decisions, and helping teams thrive. The Council oversees this work to ensure it stays aligned with Applifting's purpose, values, and long-term direction.
+
+Here's how the collaboration works:
+
+- The Council approves company strategy and can give feedback or request changes.
+- The C-Team helps shape the strategy and is responsible for fulfilling it.
+- The Council evaluates C-Team goals (C-Goals 🌊🕊️), ensuring they align with the company's direction.
+- The C-Team regularly reports on their work and results.
+- If a Council member strongly disagrees with a C-Team decision, they can request a discussion or even propose a veto — but the goal is alignment, not control.
+- The Council can propose dismissal of the C-Team if needed, and has a vote if Founders disagree.
+
+To avoid any conflict of interest, no one can be a member of both the Council and the C-Team at the same time.
+
+The Council is not a manager of the C-Team, but it is a strategic partner and ethical compass. Trust, transparency, and regular communication keep this relationship healthy.
+
+## How the Council Works
+
+- Council meets every two weeks. Any Applifter can attend, listen in, or ask for a topic to be added to the agenda (todo link to section for applifters).
+- Meetings are public unless the topic is sensitive — for example, when it concerns personal conflict, emotionally charged issues or confidential matters.
```

Comment:
Private meetings happen to discuss sensitive topicts .... and to prepare topics proposals for the council.

Tohle bych ráda upravila, protože z tohoto může znít, jako že většina práce se děje na public councilu. Ale důležité je, aby si lidé uvědomili, že se spoustu diskuzí a vyjasňování děje nejdříve na těch private councilech. Můžeme probrat kdyžtak osobně :)